pygmt/src/ternary.py

```diff
@@ -66,8 +74,21 @@ def ternary(self, data, **kwargs):
     {transparency}
     """
     kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
+
+    if alabel is None:
+        alabel = "-"
+
+    if blabel is None:
+        blabel = "-"
+
+    if clabel is None:
+        clabel = "-"
+
+    kwargs["L"] = "/".join([alabel, blabel, clabel])
```

Comment:
`alabel` should be in `str` type, but users may pass other invalid types (e.g., `int`).
Do you think we should rewrite the code to:
```
alable = str(alabel) if alable is not None else "-"
```

Reply:
Good point! Also makes the code a little bit shorter 😉